### PR TITLE
chore: merge web API fixes in master to 2.33 branch

### PR DESCRIPTION
**Pull request by larshelge**

### CODE REVIEW

File: src/commonmark/en/content/developer/web-api.md [hunk 1]
```diff
@@ -788,13 +782,13 @@ locales).
...
-version for DHIS 2.25 is *25*.
+follows the DHIS2 major version numbering. As an example, the API
+version for DHIS 2.25 is `25`.
 
 You can access a specific API version by including the version number
-after the */api* component, as an example like this:
+after the `/api` component, as an example like this:
 
-    /api/26/dataElements
+    /api/dataElements
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/developer/web-api.md]: The text above suggests that this should be an example _with_ a version number.

File: src/commonmark/en/content/developer/web-api.md [hunk 2]
```diff
@@ -1262,41 +1256,46 @@ Multiple transformers can be used by repeating the transformer
...
-    
-    Include an array with IDs of the organisation units:
-    /api/31/categoryOptions.json?fields=id,organisationUnits~pluck
-    
-    Include an array with the names of the organisation units (collection only returns field name):
-    /api/31/categoryOptions.json?fields=id,organisationUnits~pluck[name]
+/api/dataElementGroups?fields=id,displayName,dataElements~paging(1;20)
+
+# Include array with IDs of organisation units:
+/api/33/categoryOptions.json?fields=id,organisationUnits~pluck
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/developer/web-api.md]: Why use the version number here, when it is removed elsewhere? (same question for 3 lines below)

File: src/commonmark/en/content/developer/web-api.md [hunk 3]
```diff
@@ -6149,61 +6289,63 @@ of the approval levels. For example:
...
-| :- | :--- |
-| aoc | Attribute option combination identifier |
-| pe | Period identifier |
-| ou | Organisation Unit identifier |
-| permissions | The permissions: 'mayApprove', 'mayUnapprove', 'mayAccept', 'mayUnaccept', and 'mayReadData' (same definitions as for get single approval status.) |
-| state | One of the data approval states (same as for get single approval status.) |
+```json
+[
+  {
+    aoc: "HllvX50cXC0",
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/developer/web-api.md]: name strings should be wrapped in double quotes for correct json format.

File: src/commonmark/en/content/developer/web-api.md [hunk 4]
```diff
@@ -13094,28 +13352,30 @@ messages.
...
+```
 
-    {
-      "message":"Sms Text",
-      "recipients": [
-        "47XXXXXX1",
-        "47XXXXXX2"
-      ]
-    }
+> ***Note***
```
Comment (Philip-Larsen-Donnelly) [inline src/commonmark/en/content/developer/web-api.md]: should be two asterisks each side. (I'm not sure this will render correctly)

**Review by Philip-Larsen-Donnelly — CHANGES_REQUESTED** [review]
Please see comments.
(apart from those, it looks really good!)

File: src/commonmark/en/content/developer/web-api.md (hunk 4 above)
Comment (larshelge) [reply src/commonmark/en/content/developer/web-api.md]: Fixed.

File: src/commonmark/en/content/developer/web-api.md (hunk 3 above)
Comment (larshelge) [reply src/commonmark/en/content/developer/web-api.md]: Fixed.

File: src/commonmark/en/content/developer/web-api.md (hunk 2 above)
Comment (larshelge) [reply src/commonmark/en/content/developer/web-api.md]: Fixed.

File: src/commonmark/en/content/developer/web-api.md (hunk 1 above)
Comment (larshelge) [reply src/commonmark/en/content/developer/web-api.md]: Fixed.

**Pull request merged**